src/js/rx/components/ErrorMessages.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const ErrorMessages = ({ errors }) => {
+  if (errors && errors.length > 0) {
+    return (<span>
+      <br/>
+      {
+        errors.map(e => {
+          return (
+            <span key={e.code}>
+              Error {e.code}: {e.detail}
+              <br/>
```

Comment:
ul and li are probably more semantic: I added styles so that we can use that for the error messages component